### PR TITLE
7 new stat geog registers

### PR DESCRIPTION
**Pull request by Jamie-Atkinson**
https://statistical-geography-county-eng.alpha.openregister.org/
https://statistical-geography-london-borough-eng.alpha.openregister.org/
https://statistical-geography-metropolitan-district-eng.alpha.openregister.org/
https://statistical-geography-registration-district-eng.alpha.openregister.org/
https://statistical-geography-registration-district-wls.alpha.openregister.org/
https://statistical-geography-non-metropolitan-district-eng.alpha.openregister.org/
https://statistical-geography-unitary-authority-eng.alpha.openregister.org/

### CODE REVIEW

File: data/alpha/field/statistical-geography-council-area-eng.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: statistical-geography-council-area-eng
+phase: alpha
+register: statistical-geography-council-area-eng
+text: The statistical geography code for a council area in england.
```
Comment (nacnudus) [inline data/alpha/field/statistical-geography-council-area-eng.yaml]: capital 'E' for 'England'

File: data/alpha/field/statistical-geography-county-eng.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: statistical-geography-county-eng
+phase: alpha
+register: statistical-geography-county-eng
+text: The statistical geography code for a county in england.
```
Comment (nacnudus) [inline data/alpha/field/statistical-geography-county-eng.yaml]: E

File: data/alpha/field/statistical-geography-london-borough-eng.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: statistical-geography-london-borough-eng
+phase: alpha
+register: statistical-geography-london-borough-eng
+text: The statistical geography code for a london borough in England.
```
Comment (nacnudus) [inline data/alpha/field/statistical-geography-london-borough-eng.yaml]: London

File: data/alpha/field/statistical-geography-metropolitan-council-eng.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: statistical-geography-metropolitan-council-eng
+phase: alpha
+register: statistical-geography-metropolitan-council-eng
+text: The statistical geography code for a metropolitan council in england.
```
Comment (nacnudus) [inline data/alpha/field/statistical-geography-metropolitan-council-eng.yaml]: E

File: data/alpha/field/statistical-geography-non-metropolitan-council-eng.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: statistical-geography-non-metropolitan-council-eng
+phase: alpha
+register: statistical-geography-non-metropolitan-council-eng
+text: The statistical geography code for a non metropolitan council in england.
```
Comment (nacnudus) [inline data/alpha/field/statistical-geography-non-metropolitan-council-eng.yaml]: E

File: data/alpha/field/statistical-geography-registration-district-eng.yaml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: statistical-geography-registration-district-eng
+phase: alpha
+register: statistical-geography-registration-district-eng
+text: The statistical geography code for a registration district in England.
```
Comment (nacnudus) [inline data/alpha/field/statistical-geography-registration-district-eng.yaml]: :man_dancing:

File: data/alpha/register/statistical-geography-metropolitan-district-eng.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+fields:
+- statistical-geography-metropolitan-council-eng
+- local-authority-eng
+- start-date
+- end-date
+phase: alpha
+register: statistical-geography-metropolitan-council-eng
+registry: office-for-national-statistics
+text: Statistical geography data for a metropolitan council in England
```
Comment (nacnudus) [inline data/alpha/register/statistical-geography-metropolitan-district-eng.yaml]: `for metropolitan councils in`

File: data/alpha/register/statistical-geography-london-borough-eng.yaml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+fields:
+- statistical-geography-london-borough-eng
+- local-authority-eng
+- start-date
+- end-date
+phase: alpha
+register: statistical-geography-london-borough-eng
+registry: office-for-national-statistics
+text: Statistical geography data for london boroughs in England
```
Comment (nacnudus) [inline data/alpha/register/statistical-geography-london-borough-eng.yaml]: London

File: data/alpha/register/statistical-geography-non-metropolitan-district-eng.yaml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+fields:
+- statistical-geography-non-metropolitan-council-eng
+- local-authority-eng
+- start-date
+- end-date
+phase: alpha
+register: statistical-geography-non-metropolitan-council-eng
+registry: office-for-national-statistics
+text: Statistical geography data for a non metropolitan council in England
```
Comment (nacnudus) [inline data/alpha/register/statistical-geography-non-metropolitan-district-eng.yaml]: `non-metropolitan councils in`

File: data/alpha/field/statistical-geography-registration-district-eng.yaml (hunk 6 above)
Comment (Jamie-Atkinson) [reply data/alpha/field/statistical-geography-registration-district-eng.yaml]: ?

**Pull request merged**